### PR TITLE
Submit DigiTrust proposal

### PR DESCRIPTION
**Pull request by phamdat721101**
#### Project Abstract

> Please replace these instructions with a brief description of your project summarising key points (1-2 paragraphs).
>
> If your application is a follow-up to a previous grant, please mention which one in the first line of the abstract and include a link to previous pull requests if applicable.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [x] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] Payment details have been provided (bank details via email _or_ Polkadot (USDC & USDT) address in the application).
- [x] I am aware that, in order to receive a grant, I (and the entity I represent) have to successfully complete a KYC/KYB check.
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [x] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Comment by phamdat721101** [comment]
I have read and hereby sign the Contributor License Agreement.

Phạm Quốc Đạt

Mobile phone: 0977705810



Vào Th 5, 15 thg 2, 2024 vào lúc 09:19 github-actions[bot] <
***@***.***> đã viết:

**Review by keeganquigley — CHANGES_REQUESTED** [review]
Hi @phamdat721101 thanks for the application. I have the following initial comments:

- Milestone 2 is missing delivery tables. Could you add these in?
- We typically only focus on the PoC and therefore aren't able to fund gathering feedback or beta testing. Could you remove deliverable 5?
- We don't support Solidity development, would you be willing to write them in ink! instead?

File: applications/DigiTrust.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+| 2. | Portfolio Creation and Blockchain Integration | - Design and develop the core functionality for creating and customizing investment portfolios. - Integrate blockchain technology for secure and transparent record-keeping of portfolio data. - Implement smart contracts for portfolio-related transactions and interactions.|
+| 3. | Social Interaction Features | - Implement real-time communication features using WebSocket for instant updates. - Develop commenting, liking, and sharing functionalities for portfolios. - Enable users to follow and interact with each other's portfolios.|
+| 4. | Integration with External Data Sources | - Connect the platform with relevant APIs for real-time market data. - Implement scraping tools or APIs for financial news and analysis. - Ensure seamless integration with external data sources for comprehensive market insights.|
+| 5. | Beta Testing and User Feedback | - Release a beta version of the platform to gather user feedback. - Address issues and make necessary improvements based on user testing. - Iterate on features and functionalities for a more polished user experience.|
+
+
+### Milestone 2 Example — Additional features
```
Comment (keeganquigley) [inline applications/DigiTrust.md:181]:
```suggestion
### Milestone 2 — Additional features
```

**Comment by phamdat721101** [comment]
Hi Keegan,
Thanks for your comments
I already updated my proposal, and opened a pull request. Please help me to
check it again
Thank you so much.

Phạm Quốc Đạt

Mobile phone: 0977705810



Vào Th 7, 17 thg 2, 2024 vào lúc 00:32 Keegan | W3F <
***@***.***> đã viết:

**Comment by keeganquigley** [comment]
Thanks @phamdat721101 we usually require mandatory deliverables for all milestones, could you also add those in for M2 as well?

**Comment by phamdat721101** [comment]
Sure I did it

Phạm Quốc Đạt

Mobile phone: 0977705810



Vào Th 7, 17 thg 2, 2024 vào lúc 09:09 Keegan | W3F <
***@***.***> đã viết:

**Review by keeganquigley — CHANGES_REQUESTED** [review]
Thanks @phamdat721101 however I'm still not seeing deliverables 0a. - 0e. present in M2. 

1. Can you add these in? You can skip the Docker deliverable if it doesn't apply; also just one article for M2 is fine.

2. We usually ask for mockups to be provided upfront for UI focused milestones, even if they are rough.

3. For M2, its not clear to me how the notifications are being built.

4. Curious why you want to use traditional email/password login? Seems like a wallet login would be sufficient seeing as its dealing with assets.

**Comment by phamdat721101** [comment]
Hi @keeganquigley,
I already updated the proposal:
 1 - I updated the Docker deliverable in M1 and M2
 2 - I already attached link to our mockups website, and you can check our
figma design (wireframe link) in project details
 3 - I updated more detail about how to build automatic funding
allocation and notification in milestone 2
 4 - In the first-version, we target to convert web2 users to web3
industry, so we want to use traditional login firstly, then we will update
to zk-login after having good traffic.

Please help me to check it.
Thank you.

Phạm Quốc Đạt

Mobile phone: 0977705810



Vào Th 2, 19 thg 2, 2024 vào lúc 23:38 Keegan | W3F <
***@***.***> đã viết:

**Comment by phamdat721101** [comment]
Thank you so much @Keegan

Phạm Quốc Đạt

Mobile phone: 0977705810



Vào Th 5, 22 thg 2, 2024 vào lúc 03:55 Keegan | W3F <
***@***.***> đã viết:

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the application. But why do you need to "Integrate blockchain technology" to track portfolio data? For this kind of solution no smart contract or anything similar should be needed, at least if the portfolio is based on digital assets. What kind of smart contracts have you deployed as part of your MVP?

File: applications/DigiTrust.md [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+Milestone 1: MVP Completion
+
+Overview:
+
+**DigiTrust** has successfully completed the development of its Minimum Viable Product (MVP), marking a crucial milestone in the journey toward creating a decentralized asset management platform within the decentralized financial social network.
+
+Key Achievements:
+
+**Blockchain Integration:** Smart contracts have been deployed to testnet, providing users with transparent and tamper-resistant asset management.
```
[the remark this answers is not in the document]
Comment (Luckygirl3289) [reply applications/DigiTrust.md:169]: Blockchain testnet

**Comment by keeganquigley** [comment]
pinging @phamdat721101

**Comment by phamdat721101** [comment]
> Thanks for the application. But why do you need to "Integrate blockchain technology" to track portfolio data? For this kind of solution no smart contract or anything similar should be needed, at least if the portfolio is based on digital assets. What kind of smart contracts have you deployed as part of your MVP?

File: applications/DigiTrust.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (phamdat721101) [reply applications/DigiTrust.md:169]: 1 - DigiTrust targets to deploy on **Statemint** parachain
2 - We are starting to work with Ink! and Substrate now, and we will migrate system from solidity base
3 - These are extra features in the second milestone, and we will explain more detail after having traffic from first-version for building portfolio management 
4 - DigiTrust is deployed in Klaytn-testnet : 0xa39c2b7957496d28cd6b3fd005b4a1584abe0c30

**Comment by phamdat721101** [comment]
> Thanks for the application. But why do you need to "Integrate blockchain technology" to track portfolio data? For this kind of solution no smart contract or anything similar should be needed, at least if the portfolio is based on digital assets. What kind of smart contracts have you deployed as part of your MVP?

Hi, DigiTrust target to track portfolio data as the post content for decentralized social network feature, which can support investors or managers to share their performance, 
We already deployed test-version in Klaytn-testnet in 0xa39c2b7957496d28cd6b3fd005b4a1584abe0c30

**Comment by keeganquigley** [comment]
Hi @phamdat721101 thank you for all the time and effort you put into the application and for your communication during the process. 

Unfortunately, the committee unanimously decided today not to move forward with the application. This is mainly due to the concerns listed above. Since you already have all the smart contracts built and it is meant to integrate with popular EVM DeFi apps, it might make more sense to migrate the contracts to an EVM-compatible chain like Moonbeam or Astar instead of re-writing them all.

Either way, I appreciate your interest in the W3F grants program and wish you the best of luck in finding funding.

**Comment by phamdat721101** [comment]
No worries, thanks for the update!

Phạm Quốc Đạt

Mobile phone: 0977705810



Vào Th 5, 7 thg 3, 2024 vào lúc 07:22 Keegan | W3F <
***@***.***> đã viết:

**Pull request closed without merging**